libs/luci-lib-nixio/src/tls-context.c

```diff
@@ -65,9 +65,8 @@ static int nixio_tls_ctx(lua_State * L) {
 		return luaL_error(L, "unable to create TLS context");
 	}
 
-#ifdef WITH_CYASSL
+	/* Setting do not verify certificate */
 	SSL_CTX_set_verify(*ctx, SSL_VERIFY_NONE, NULL);
```

Comment:
You should set the minimum version here, like you had done before.

Reply:
I'm not sure if I should drop support for protocols lower than TLS 1.2 outright, but I've added it back now.